src/object/cli_obj.c

```diff
@@ -1230,12 +1246,102 @@ obj_req_tgts_dump(struct obj_req_tgts *req_tgts)
 	}
 }
 
+static int
+obj_ec_check_exist_tgts_get(struct dc_object *obj, uint32_t map_ver, uint32_t start_shard,
```

Comment:
This looks like obj_ec_list_get_shard or query_key()?

Reply:
with some similar but not same.

Reply:
is this just "choosing the available parities, otherwise all data shards.".   or I miss something?